src/OrbitBase/TemporaryFile.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitBase/TemporaryFile.h"
+
+#include <stdlib.h>
+
+namespace orbit_base {
+
+void TemporaryFile::Create() noexcept {
+  std::error_code error;
+  std::filesystem::path temporary_dir = std::filesystem::temp_directory_path(error);
+  std::string file_path = (temporary_dir / "orbit_XXXXXX").string();
```

Comment:
Move this after `if (error)`, it's not used before that.

Reply:
done.